projector-server/src/main/kotlin/org/jetbrains/projector/server/HeadlessInitializer.kt

```diff
@@ -76,7 +62,6 @@ internal fun setupRepaintManager() {
 internal fun setupSystemProperties() {
   // Setting these properties as run arguments isn't enough because they can be overwritten by JVM
   System.setProperty(ENABLE_PROPERTY_NAME, true.toString())
-  System.setProperty("java.awt.graphicsenv", PGraphicsEnvironment::class.java.canonicalName)
```

Comment:
Why do we remove that?

Reply:
This property is removed in jdk17 and has no effect. Graphics environment is now injected via `IjAwtTransformer` for both jdk11 and jdk17